eva_submission/eload_backlog.py

```diff
@@ -145,7 +145,10 @@ def copy_valid_config_to_brokering_after_merge(self):
         Convert the valid entry in the config to the one that should be created after brokering.
         This is only useful after we merged the vcf files otherwise this function will not do anything.
         """
-        for analysis_alias, analysis_data in self.eload_cfg.query('validation', 'valid', 'analyses'):
+        if not self.eload_cfg.query('validation', 'valid', 'analyses'):
+            self.error('Merge did not complete Most likely because one of the validation did not pass.')
```

Comment:
```suggestion
            self.error('Merge did not complete, most likely because one of the validation did not pass.')
```

Also, should this raise an exception or is it fine with just the error message?

Reply:
Good idea. I've changed it.